code/_onclick/item_attack.dm

```diff
@@ -71,6 +71,7 @@ avoid code duplication. This includes items that may sometimes act as a standard
 		if(prob(10))
 			for(var/mob/living/carbon/human/H in viewers(user))
 				SEND_SIGNAL(H, SWORD_OF_TRUTH_OF_DESTRUCTION, src)
+				eotp.addObservation(200)
```

Comment:
```suggestion
				if(eotp)
					eotp.addObservation(200)
```

Reply:
I'll add that in when I get a chance. 
Even still, last checked, observation only affects power accumulation for the Eye. If they eye were to be destroyed, then observation wouldn't affect anything. Wouldn't that make this change moot?